packages/ra-core/src/form/validate.ts

```diff
@@ -1,4 +1,5 @@
 import lodashMemoize from 'lodash/memoize';
+import format from 'date-fns/format';
```

Comment:
Browsers have an API for formatting dates (Intl), no need for an external library

Reply:
> Hi, these validators look great 👍
> 
> Thanks for contributing 😊.
> 
> I've one remark: Can we can consider using another name for these validators?
> 
> * Minimal date means the input should contain a date after the specified one
> * Maximal date means the input should contain a date before the specified one
> 
> So maybe you can name these methods `afterDate` and `beforeDate`. What do you think?



> Hi, these validators look great 👍
> 
> Thanks for contributing 😊.
> 
> I've one remark: Can we can consider using another name for these validators?
> 
> * Minimal date means the input should contain a date after the specified one
> * Maximal date means the input should contain a date before the specified one
> 
> So maybe you can name these methods `afterDate` and `beforeDate`. What do you think?

I read your comment. Thank you for reviewing my code!
I implemented code to allow the base date to be included.
You means that it'll be better if the code excludes the base date? or just changes the validator name?

Reply:
> Browsers have an API for formatting dates (Intl), no need for an external library

Thank you! 
I used `new Intl.DateTimeFormat().format(date)` also changed format options (DateTimeFormatOptions).
The option can be a little long like this. I don't know this is good. 
```
const options = {
    year: 'numeric',
    month: 'short',
    day: 'numeric',
    hour: 'numeric',
    minute: 'numeric',
};
```
It needs to know `DateTimeFormatOptions` and  readability doesn't look good. 
What do you think?